README.md

```diff
@@ -10,7 +10,7 @@ This monorepo will eventually contain all civ projects and development
 A docker compose stack is provided to help test containers built from
 this repo. To start the stack, run the following commands:
 
-1. `gradle :ansible:build`
+1. `./gradlew :ansible:build`
```

Comment:
This is only the case for linux/unix, should be `.\gradlew.bat build` for PowerShell and I believe `gradlew.bat build` for cmd on windows?

Reply:
Sorry, missed this. Powershell and CMD on Windows 10 both accept `.\gradlew`